config/config.json.example

```diff
@@ -0,0 +1,54 @@
+{
+  "chains": [
+    {
+      "name": "eth",
+      "type": "ethereum",
+      "id": "0",
+      "endpoint": "https://goerli.infura.io/v3/178b87e860404572bc217ca53fe594bd",
```

Comment:
Public URL should be used. Please, update

Reply:
fixed

Reply:
I still see old link

Reply:
Fixed